core/storage/user/gae_models.py

```diff
@@ -195,6 +195,25 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.DELETE
 
+    @staticmethod
+    def export_data(user_id):
+        """(Takeout) Export CompletedActivitiesModel's user properties.
+        Args:
+            user_id: str. The user_id denotes which user's data to extract.
+        Returns:
+            dict or None. A dict containing the user-relevant properties of
```

Comment:
Here and elsewhere, specify these in more detail. E.g.:

    A dict with two keys, 'completed_exploration_ids' and 'completed_collection_ids'. The corresponding values are lists of the IDs of the explorations and collections, respectively, which the given user has completed.

Reply:
Ack